CLogger/include/CLogBuilder/CLogBuilder.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+#include "stdafx.h"
+
+#include <memory>
+#include <list>
+
+#include "ELogConfig/ELogConfig.h"
+#include "CLogger/CLogger.h"
+#include "ELogLevel/ELogLevel.h"
+
+/// <summary>
+///		Namespace, that contains all classes and functions
+///		to work with CLogger and its macro-functions
+/// </summary>
+namespace Log
+{
+	/// <summary>
+	///		Class, that sets basic configuration of <c>CLogger</c> and creates it
+	/// </summary>
+	class CLOGGER_API CLogBuilder {
+	public:
+		CLogBuilder() = delete;
+		explicit CLogBuilder(const std::string& log_name, ELogLevel log_level);
```

Comment:
use explicit only for constructors with 1 parameter

Reply:
Are we not afraid about smth like: `{ std::string("TestCtor"), ELogLevel::TRACE  }`, that will call implicit ctor of `CLogBuilder`?

Reply:
Yes you right, we should use explicit for ctors with multiple params since C++11.